packages/credentials/src/status-list-credential.ts

```diff
@@ -0,0 +1,238 @@
+import pako from 'pako';
+import { getCurrentXmlSchema112Timestamp } from './utils.js';
+import { VerifiableCredential, DEFAULT_VC_CONTEXT, DEFAULT_VC_TYPE, VcDataModel } from './verifiable-credential.js';
+import type { ICredentialStatus} from '@sphereon/ssi-types';
+import { Convert } from '@web5/common';
+
+export const DEFAULT_STATUS_LIST_VC_CONTEXT = 'https://w3id.org/vc/status-list/2021/v1';
+export const DEFAULT_STATUS_LIST_VC_TYPE = 'StatusList2021Credential';
+
+export enum StatusPurpose {
+  REVOCATION = 'revocation',
+  SUSPENSION = 'suspension',
```

Comment:
Unconventional to use all caps for enum values, consider:
```suggestion
  Revocation = 'Revocation',
  Suspension = 'Suspension',
```

or if these values are predefined to be lower case per spec:

```suggestion
  revocation = 'revocation',
  suspension = 'suspension',
```

Reply:
Handled in https://github.com/TBD54566975/web5-js/pull/486.